streams/src/main/java/org/apache/kafka/streams/state/ReadOnlyWindowStore.java

```diff
@@ -33,11 +33,11 @@
     /**
      * Get the value of key from a window.
      *
-     * @param key       the key to fetch
-     * @param time      start timestamp (inclusive) of the window
+     * @param key  the key to fetch
+     * @param time start timestamp (inclusive) of the window
      * @return The value or {@code null} if no value is found in the window
      * @throws InvalidStateStoreException if the store is not initialized
-     * @throws NullPointerException If {@code null} is used for any key.
+     * @throws NullPointerException       If {@code null} is used for any key.
```

Comment:
nit: is this intentional? Also I'd suggest we do not use capitalized `If` to be consistent with the above line, ditto elsewhere below.

Reply:
yes, IDE reformatting. I've fixed the capitalized `If`